profile2/impl/src/java/org/sakaiproject/profile2/dao/impl/ProfileDaoImpl.java

```diff
@@ -88,6 +88,26 @@
 	  	
 	  	return getHibernateTemplate().execute(hcb);
 	}
+
+	/**
+ 	 * {@inheritDoc}
+ 	 */
+	public List<String> getOutgoingConnectionUserIdsForUser(final String userId) {
+
+		//get friends of this user [and map it automatically to the Friend object]
+		//updated: now just returns a List of Strings
+		HibernateCallback<List<String>> hcb = session -> {
+            Query q = session.getNamedQuery(QUERY_GET_OUTGOING_FRIEND_REQUESTS_FOR_USER);
+            q.setParameter(USER_UUID, userId, StringType.INSTANCE);
```

Comment:
There is a setString() which is more appropriate than setParameter and specifying StringType